examples/reference/widgets/Tabulator.ipynb

```diff
@@ -0,0 +1,724 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from bokeh.models.widgets.tables import NumberFormatter, BooleanFormatter\n",
+    "\n",
+    "bokeh_formatters = {\n",
+    "    'float': NumberFormatter(format='0.00000'),\n",
+    "    'bool': BooleanFormatter(),\n",
+    "}\n",
+    "\n",
+    "pn.widgets.Tabulator(df, formatters=bokeh_formatters)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The list of valid Bokeh formatters includes:\n",
+    "    \n",
+    "* [BooleanFormatter](https://docs.bokeh.org/en/latest/docs/reference/models/widgets.tables.html#bokeh.models.widgets.tables.BooleanFormatter)\n",
+    "* [DateFormatter](https://docs.bokeh.org/en/latest/docs/reference/models/widgets.tables.html#bokeh.models.widgets.tables.DateFormatter)\n",
+    "* [NumberFormatter](https://docs.bokeh.org/en/latest/docs/reference/models/widgets.tables.html#bokeh.models.widgets.tables.NumberFormatter)\n",
+    "* [HTMLTemplateFormatter](https://docs.bokeh.org/en/latest/docs/reference/models/widgets.tables.html#bokeh.models.widgets.tables.HTMLTemplateFormatter)\n",
+    "* [StringFormatter](https://docs.bokeh.org/en/latest/docs/reference/models/widgets.tables.html#bokeh.models.widgets.tables.StringFormatter)\n",
+    "* [ScientificFormatter](https://docs.bokeh.org/en/latest/docs/reference/models/widgets.tables.html#bokeh.models.widgets.tables.ScientificFormatter)\n",
+    "\n",
+    "However in addition to the formatters exposed by Bokeh it is also possible to provide valid formatters built into the Tabulator library. These may be defined either as a string or as a dictionary declaring the 'type' and other arguments, which are passed to Tabulator as the `formatterParams`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tabulator_formatters = {\n",
+    "    'int': 'start',\n",
+    "    'float': {'type': 'progress', 'max': 10},\n",
+    "    'bool': {'type': 'tickCross'}\n",
+    "}\n",
+    "\n",
+    "pn.widgets.Tabulator(df, formatters=tabulator_formatters)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The list of valid Tabulator formatters can be found in the [Tabulator documentation](http://tabulator.info/docs/4.9/format#format-builtin).\n",
+    "\n",
+    "## Editors\n",
+    "\n",
+    "Just like the formatters the `Tabulator` will natively understand the Bokeh `Editor` types, however in the background it will replace most of them with equivalent editors natively supported by the tabulator library:"
```

Comment:
```suggestion
    "Just like the formatters, the `Tabulator` will natively understand the Bokeh `Editor` types. However, in the background it will replace most of them with equivalent editors natively supported by the tabulator library:"
```